scripts/check_coverage.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+CHECKCOV=false
+
+#cleanup
+finish() {
+  cat scripts/results.tmp
+  if [ -e "scripts/results.tmp" ]; then
+    rm scripts/results.tmp
+  fi
+
+  if [ -d ".cover" ]; then
+    rm -rf .cover
+  fi
+}
+
+trap finish EXIT ERR INT TERM
+
+#start script logic
+./scripts/coverage.sh > scripts/results.tmp
```

Comment:
any reason this tmp file is in `scripts`, rather than `.cover`?

Not a big deal... just wondering, cause it seems out of place and complicates things a bit.

Reply:
I've been playing around with this part of the code to have the final results.tmp file located in the same folder with the rest of the package coverage files (ie .cover).  When the main script coverage script (ie coverage.sh) runs, it recreates the .cover directory so that it contains data relevant to that run.  When I try to pipe the results from coverage.sh to a file within this .cover directory, it gets deleted when the directory is recreated (since the results.tmp is created first to store the results before the script runs).  

So to maintain the output in the results.tmp file for the second half of the check_coverage.sh script, the results.tmp file has to be located in a different directory than where the rest of the coverage data sits.

If you have any thoughts/suggestions around this to try out, please let me know.